Add connect wallet with browser example using Next.js (#8)
metaplex-foundation/js-examples: connect-wallet/README.md

# Connect Wallet in the Browser

> This example sets up a new React app with Metaplex using Next.js.
> 
> This tutorial shows how to connect your wallet to the browser via [Solana Wallet Adapter Library](https://github.com/solana-labs/wallet-adapter), and use the [Metaplex JS SDK](https://github.com/metaplex-foundation/js) to display a random NFT from the wallet and refresh it when a button is clicked.

Reviewer: ```suggestion
In this tutorial, we will see how to connect a wallet in the browser using the [Solana Wallet Adapter Library](https://github.com/solana-labs/wallet-adapter) and the [Metaplex JS SDK](https://github.com/metaplex-foundation/js).

Once the user has connected their wallet, we will display a random NFT from their wallet and refresh it at the click of a button.
```

This example has been generated using the following steps:

> 1. **Create a new project using Next.js.**
>     Redirect to [this](https://github.com/metaplex-foundation/js-examples/tree/main/getting-started-nextjs) link to know how to create next.js application and install metaplex successfully.

Reviewer: ```suggestion
1. **Create a new project using Next.js.**

   Let's start by spinning off a Next.js app with the Metaplex JS SDK installed. You may achieve this by [following this tutorial](https://github.com/metaplex-foundation/js-examples/tree/main/getting-started-nextjs).
```

> 2. **Install the needed dependencies.**

Reviewer: ```suggestion
2. **Install the wallet adapter libraries.**
```

   ```sh
    npm install @solana/wallet-adapter-base \
         @solana/wallet-adapter-react \
         @solana/wallet-adapter-react-ui \
         @solana/wallet-adapter-wallets
   ```

3. **Create the script files and copy the code from the corresponding file in the repository**
   ```
      cd pages
      touch ShowNFTs.js MetaplexProvider.js useMetaplex.js

>    ```

Reviewer: Could you please split that step into three steps, to explain what each file does in more detail? It would be great to have some code snippets here as well so the reader can quickly grasp what they need to do.

4. **That's it!** 🎉

>    You're now ready to start building your app. You can use the following commands to build and serve your app.
> 
>    ```sh
>    # Build and serve for development.
>    npm run dev
> 
>    # Build and serve for production.
>    npm run build && npm run start
>    ```

Reviewer: Let's make the final step more tailored for this tutorial. Could you also add a few screenshots of the final result? Maybe one with the wallet modal and one that says the wallet has been connected?

```suggestion
   You're now ready to start building your app whilst having access to the user's wallet!
   
   Note that now that the wallet has been integrated with the Metaplex JS SDK, you can use all of its features on behalf of the user and it will request their approval every time they need to send a transaction or sign a message.
   
   Let's see a few screenshots of the final result!
```

> 
>    If you're interested in how this example app is using the Metaplex SDK, check out the [`index.js`](./pages/index.js), [`ShowNFTs.js`](./pages/ShowNFTs.js), [`MetaplexProvider.js`](./pages/MetaplexProvider.js), [`useMetaplex.js`](./pages/useMetaplex.js) and [`Home.module.css`](./styles/Home.module.css) files in the `pages` and `styles` directories.

Reviewer: We can remove this because step 3 is going to be broken down into more detailed steps that will reference these files.

```suggestion
```
